### PR TITLE
Release v1.0.1 (ETH-1063)

### PR DESCRIPTION
**Pull request by nmvalera**
## Description

Release Liquid Collective v1.0.1

Contracts deployed

- River
- OperatorsRegistry

## Pull Request Type

- [ ] 💫 New Feature (Breaking Change)
- [ ] 💫 New Feature (Non-breaking Change)
- [ ] 🛠️ Bug fix (Non-breaking Change: Fixes an issue)
- [x] 🕹️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)

## Additional comments

Solidity compiler optimization parameters have been changed so River smart contract respects Ethereum max contract size limits.

### CODE REVIEW

**Review by 0xvv — DISMISSED** [review]
looks good to me

File: deploy/mainnet/11_deploy_v1.0.1.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { isDeployed, logStep, logStepEnd } from "../../ts-utils/helpers/index";
+
+const func: DeployFunction = async function ({ deployments, network, getNamedAccounts }: HardhatRuntimeEnvironment) {
+  if (!["mainnet", "hardhat"].includes(network.name)) {
+    throw new Error("Invalid network for mainnet deployment");
+  }
+
+  const { deployer } = await getNamedAccounts();
+
+  // await deployments.deploy("RiverV1_Implementation_1_0_1", {
```
Comment (mortimr) [inline deploy/mainnet/11_deploy_v1.0.1.ts:12]: why is this block commented ? it seems this contract was deployed by looking at the generated artifacts

**Review by mortimr — APPROVED** [review]
LGTM but just a question about a commented block in a migration script

File: deploy/mainnet/11_deploy_v1.0.1.ts (hunk 1 above)
Comment (nmvalera) [reply deploy/mainnet/11_deploy_v1.0.1.ts]: Uncommented. Thanks

**Review by 0xvv — APPROVED** [review]

**Review by mortimr — APPROVED** [review]
LGTM !

**Pull request merged**